congressional-support.yaml

```diff
@@ -0,0 +1,119 @@
+- type: house
+  name: House Documents Repository
+  url: http://docs.house.gov/
```

Comment:
All the `url` fields will need to be quoted, because they contain a colon themselves.